_data/toc.yaml

```diff
@@ -1070,14 +1033,64 @@ reference:
         title: docker volume rm
     - path: /engine/reference/commandline/wait/
       title: docker wait
+  - sectiontitle: Compose (docker-compose) CLI reference
```

Comment:
Can we avoid repeating Compose here? How about 'docker compose reference`?

Reply:
good idea. I changed it to Docker Compose CLI.